ui/eureka/eureka.js

```diff
@@ -1949,13 +1590,38 @@ let gWeatherIcons = {
 let gNightIcon = '&#x1F319;';
 let gDayIcon = '&#x263C;';
 
+class EventQueue {
```

Comment:
I kind of think this is a little bit overkill.  I think you could just make fatequeue a `[]` that you push and shift from and save twenty lines of code.

Reply:
It was a bit overkill, changed. 👍